backend/canvas_app_explorer/migrations/0009_auto_20220623_1526.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.13 on 2022-06-23 15:26
+
+# Resource(s):
+# - https://docs.djangoproject.com/en/4.0/topics/migrations/#data-migrations
+# - https://lti.colcampus.com/xml_builder
+
+from django.db import migrations
+
+
+PLACEMENT_NAMES = [
```

Comment:
can we provide a source (from IMS, Canvas, etc) for this list of placement options? I am not sure whether[ this list in the Canvas LTI course](https://canvas.instructure.com/courses/913512/pages/placements) is up-to-date.

Reply:
Perhaps we can come up with a more refined list from this? https://canvas.instructure.com/doc/api/file.tools_intro.html

Reply:
The Canvas API doc page should be good.

Reply:
Okay, I updated this list and the link. I took out ones if I didn't see any indication that the were supported. Maybe some of them are for us or in Canvas Commons, per the LTI doc @zqian shared before?
```
'Assignment Configuration',
'Assignment Menu',
'Course Home Sub Navigation',
'Course Settings Sub Navigation',
'Discussion Menu',
'File Menu',
'Global Navigation',
'Module Menu',
'Post Grades',
'Quiz Menu',
'Tool Configuration',
'Wiki Page Menu'
```

Reply:
One of the reasons I opted to make these configurable verses having an internal list was because Canvas seems to add these on occasion and promotes them to being "officially documented" and used. I'm not sure how often this happens though.